spec/models/custom_tab_spec.rb

```diff
@@ -6,4 +6,13 @@
   describe 'associations' do
     it { is_expected.to belong_to(:enterprise).required }
   end
+
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:title) }
+
+    it "should not validate title if length is greater than 20" do
+      custom_tab = build(:custom_tab, title: "a" * 21)
+      expect(custom_tab).to be_invalid
+    end
```

Comment:
https://matchers.shoulda.io/docs/v5.3.0/Shoulda/Matchers/ActiveModel.html#validate_length_of-instance_method
```suggestion
    it { is_expected.to validate_presence_of(:title) }

    it { is_expected.to validate_length_of(:title).is_at_most(20) }
```